project/BRANCHES-AND-TAGS.md

```diff
@@ -0,0 +1,35 @@
+Branches and tags
+=================
+
+Note: details of the release process for the Engine are documented in the
+[RELEASE-CHECKLIST](https://github.com/docker/docker/blob/master/project/RELEASE-CHECKLIST.md).
+
+# Branches
+
+Docker projects should normally have only three living branches at all time, including the regular
+`master` branch:
+
+## `docs` branch
+
+The `docs` branch support documentation updates between product releases. This branch allow us to
+decouple documentation releases from product releases.
+
+## `release` branch
+
+The `release` branch contains the last _released_ version of the code for the project.
+
+The `release` branch is only updated at each public release of the project. The mechanism for this
+is that the release is materialized by a pull request against the `release` branch which lives for
+the duration of the code freeze period. When this pull request is merged, the `release` branch gets
+updated, and its new state is tagged accordingly.
+
+# Tags
+
+Any public release of a compiled binary, with the logical exception of nightly builds, should have
```

Comment:
Yeah, I tried to get the maintainers to get PGP keys set up a good long while back so we could start signing releases this way (because verifying binaries via automated processes is a huge +1 in my book -- there are several places I use the raw binaries that I'd love to be able to verify them), but the take rate was low.

All that aside, I think this would be much easier to manage if we had an official "release server" though (like we've been discussing) so that these keys don't have to be shared around more than the official releasing server, and so people don't have to import every single maintainer's personal keys to verify the binaries (or find out which maintainer was the "release maintainer" for a particular release).  Python signs their artifacts with the personal key of the release maintainer and it's a real PITA to try and find which key signed which release.

TL;DR, I'm not opposed, but I don't know that just having the release manager use their own personal key is really tenable, and sharing keys isn't really great either.